src/cli.rs

```diff
@@ -237,11 +244,25 @@ pub fn process_command_line_arguments<'a>(
         None => Some(available_terminal_width),
     };
 
+    let paging_mode = match opt.paging_mode.as_ref() {
+        "always" => PagingMode::Always,
+        "never" => PagingMode::Never,
+        "auto" => PagingMode::QuitIfOneScreen,
+        _ => {
+            eprintln!(
+                "Invalid paging value: {} (valid values are \"always\", \"never\", and \"auto\")",
+                opt.paging_mode
+            );
+            process::exit(1);
+        }
+    };
+
```

Comment:
I made a small change here to demand that if the user supplies a value, it is valid.